core/templates/components/state-editor/state-hints-editor/state-hints-editor.component.html

```diff
@@ -3,7 +3,7 @@
     <mat-card class="oppia-editor-card-with-avatar oppia-mobile-collapsible-card">
       <div class="oppia-editor-card-body">
         <div class="state-hints-header-container oppia-mobile-collapsible-card-header" (click)="toggleHintCard()">
-          <div class="state-hints-header" *ngIf="editabilityService.isEditableOutsideTutorialMode()">
```

Comment:
I had to remove the `ngIf` statement from here because it was causing the 'Hints' header to disappear when the user viewed their question submission history.
![image](https://user-images.githubusercontent.com/73544247/179847434-234ace9b-ee7c-45ff-95ed-1d110557be26.png)

Reply:
Does removing the ngIf have any other side effects? E.g. when hints are not supposed to be editable, it is not editable.

Reply:
No, the actual 'add hint icon' and the 'add hint button' are still only accessible if the hints are editable. 
(line 16)
![image](https://user-images.githubusercontent.com/73544247/180623601-1c518a7d-ac10-4183-9f24-bd9fe69d8ade.png)
(line 10)
![image](https://user-images.githubusercontent.com/73544247/180623683-0381a043-dbcd-4872-af08-4e3370bb8dcb.png)